### PR TITLE
LINK-2031 vulnerable dependencies

### PR DESCRIPTION
**Pull request by mikkojamG**
## Description :sparkles:

Fixing vulnerabilities and removing deprecated packages.

### Closes :no_good_woman:

**[LINK-2031](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2031):**

## Additional notes :spiral_notepad:

[LINK-2031]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2031?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr404-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr404-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr404-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr404-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr404-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr404-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by Riippi** [comment]
```
error vite-bundle-visualizer@1.2.1: The engine "node" is incompatible with this module. Expected version "^18.19.0 || >=20.6.0". Got "18.18.2"
error Found incompatible module.
```
I got that on yarn install.
.nvmrc should be updated?

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr404-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr404-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by Riippi — APPROVED** [review]

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v18
\ No newline at end of file
+v18.19.0
```
Comment (jorilindell) [inline .nvmrc:1]: Not really related to this PR but it could be good idea upgrade to node version 20 at some point

**Review by jorilindell — APPROVED** [review]
Looks good

**Pull request merged**